tests/EStimLibrary.UnitTests/Core/UtilsTests.cs

```diff
@@ -1,123 +1,212 @@
 ﻿using EStimLibrary.Core;
 
-
 namespace EStimLibrary.UnitTests.Core;
 
-
+/// <summary>
+/// Unit tests for the utility methods in the Utils class.
+/// </summary>
 public class UtilsTests
 {
+    /// <summary>
+    /// Tests that GetAvailableTypes returns types assignable from the specified base type.
+    /// </summary>
     [Fact]
     public void GetAvailableTypes_ShouldReturnTypes_WhenTypeExists()
```

Comment:
This is a style note, but for convention, the "ShouldXXX" component of test method names comes last. Convention is <MethodTested>_<Scenario>_Should<ExpectedOutput>()